src/crafting_gui.cpp

```diff
@@ -669,6 +709,23 @@ const recipe *select_crafting_recipe( int &batch_size )
     return chosen;
 }
 
+static bool query_is_yes( std::string query )
+{
+    std::string subquery = query.substr( 2 );
+
+    if( ( subquery == "yes" ) || ( subquery == "y" ) || ( subquery == "1" ) ||
+        ( subquery == "true" ) || ( subquery == "t" ) || ( subquery == "on" ) ) {
+        return true;
+    }
+    return false;
```

Comment:
The condition already yields `bool`: there's no need to write it like: `if (bool) true else false`. Also, the parentheses are excessive. Simply write:
```c++
return 
    subquery == "yes" ||
    subquery == "y" ||
...
```